src/lifx.d.ts

```diff
@@ -1,367 +1,328 @@
-export class Client {
+import {RemoteInfo} from 'dgram';
+import {EventEmitter} from 'eventemitter3';
+import {AddressInfo} from 'net';
+
+export class ClientOptions {
+  /** The IPv4 address to bind to  */
```

Comment:
At least WebStorm doesn't seem to show these when I hover over `client.init` so these aren't much of use here. See my comments on the client interface.

Reply:
They should. It does in vscode.